### PR TITLE
Update working group description.

### PR DESCRIPTION
**Pull request by shamil-gadelshin**
Added extrinsics description:
- set budget
- leader spending
- set status text

### CODE REVIEW

File: governance/working-groups.md [hunk 1]
```diff
@@ -363,5 +363,55 @@ Staking lock is increased by `stake_amount`.
...
+* A lead worker is set.
+* A caller must be an active leader.
+* `amount` is greater than zero.
+* The working group budget is sufficient to transfer the tokens.
+
+#### Effect
+
+Account balance is increased by `amount`.
+
+### Setting the working group budget
```
Comment (bedeho) [inline governance/working-groups.md:385]: The description level targeted in the handbook is functinal, not at the module level, hence I think we dont explain things at the abstraction level of public module methods or root origin gate extrinsics. Instead we would here explain this as a proposal in the proposal section of the handbook, could you do that?

File: governance/working-groups.md [hunk 2]
```diff
@@ -363,5 +363,55 @@ Staking lock is increased by `stake_amount`.
...
+
+#### Conditions
+
+* Caller must be sudo.
+
+#### Effect
+
+Working group balance is set to the new `amount`.
+
+### Setting the working group status text
```
Comment (bedeho) [inline governance/working-groups.md:401]: Can you also add this as a concept in the concept section, and also mention this as one of the activities that fall within the domain of responsibility of the lead in the `Roles` section.

**Review by bedeho — CHANGES_REQUESTED** [review]

File: governance/working-groups.md (hunk 2 above)
Comment (shamil-gadelshin) [reply governance/working-groups.md:401]: Updated the Leader role description.

File: governance/working-groups.md (hunk 1 above)
Comment (shamil-gadelshin) [reply governance/working-groups.md:385]: I added the `Set Working Group Budget` description to the handbook (which is yet to be implemented).

**Review by bedeho — APPROVED** [review]

**Pull request merged**